packages/kbn-telemetry-tools/src/tools/tasks/check_matching_schemas_task.ts

```diff
@@ -31,8 +31,16 @@ export function checkMatchingSchemasTask({ roots }: TaskContext) {
 
       if (root.parsedCollections) {
         const differences = checkMatchingMapping(root.parsedCollections, esMapping);
-
         root.esMappingDiffs = Object.keys(differences);
+        if (root.esMappingDiffs.length && throwOnDiff) {
+          throw Error(
+            `The following changes must be persisted in ${fullPath} file. Use '--fix' to update.\n${JSON.stringify(
+              differences,
+              null,
+              2
+            )}`
+          );
+        }
```

Comment:
I've got the same question as in https://github.com/elastic/kibana/pull/75747#discussion_r475635788: Are devs in charge of running these changes themselves (maintaining the merged-schemas JSON files) and we simply accept them? Or are we supposed to run the _extract_ after FF to see all the changes at once?

Reply:
Devs run the --fix to update the json files, and we approve them since we are the owners of that file. After FF we run the extract to grab json file to see all the additions between releases similiar to what we do with i18n.

Reply:
Does extract generate another JSON file? I thought it was the same one. But it that's the intended process. LGTM 👍